File: 2024_java_basics/08-conditional-statements-advanced-lab/tasks/01-cinema.java
import java.util.Scanner;

public class Cinema {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String ticket = scanner.nextLine();
        int r = Integer.parseInt(scanner.nextLine());
        int c = Integer.parseInt(scanner.nextLine());
        double income = 0.0;

        switch (ticket) {
            case "Premiere":
                income = r * c * 12.00;
                break;

            case "Normal":
                income = r * c * 7.50;
                break;

            case "Discount":
                income = r * c * 5.00;
                break;

            default:
                System.out.println("Invalid ticket type");
                scanner.close();
                return;
        }

        if (income > 0) {
            System.out.printf("%.2f leva\n", income);
        }

        scanner.close();
    }
}

File: 2022_c#_basics/08-advanced-conditional-statements-exercise/01-cinema.cs
using System;

namespace _01_Cinema
{
    internal class Program
    {
        static void Main(string[] args)
        {
            string ticket = Console.ReadLine();
            int r = int.Parse(Console.ReadLine());
            int c = int.Parse(Console.ReadLine());
            double income = 0.0;

            switch (ticket)
            {
                case "Premiere":
                    income = r * c * 12.00;
                    break;

                case "Normal":
                    income = r * c * 7.50;
                    break;

                case "Discount":
                    income = r * c * 5.00;
                    break;
            }
            if (income > 0)
            {
                Console.WriteLine($"{income:f2} leva");
            }
        }
    }
}
